### DISCUSSION TITLE
Dark reader makes websites using fomantic UI extremely slow to load

### DISCUSSION THREAD

**Discussion by HugoKirch** [darkreader/darkreader#10402] [Q&A] [answered]
Hi everyone,

I have developed a very tiny static website using fomantic UI in order to teach pygame to other students at my university.

The problem is : when I load the page while dark reader is activated, the page takes a really long time to load. When I deactivate it, it loads normally.

I've encountered the problem on the website when hosted on my VPS and when i open the HTML file locally with my navigator.

I'm using Dark Reader 4.9.60, Brave 1.45.127 and fomantic UI 2.8.8. My OS is Manjaro Linux.

You can access the website with this URL : [https://ankidets.fr/pygame](url)

Does someone have experienced something similar ? And does someone know how to fix this ?

PS: Sorry for my bad english and probably bad post, this is the first time I post all time.

**Comment by Gusted** [comment, marked as answer]
Please open a [performance issue](https://github.com/darkreader/darkreader/issues/new?assignees=&labels=Performance+Issue&template=performance-issue-report.yml&title=%5BPerformance+Issue%5D+Replace+with+title) for this

**Reply by HugoKirch** [reply]
Thanks,
I didn't check if an issue was already open, sorry
